tests/cli/test_cli.py

```diff
@@ -1060,6 +1060,11 @@ def test(ctx):
         '3', _run_command([cl, 'uls', '.time_used_less_than=' + '1%', '-f', 'parallel_run_quota'])
     )
 
+    # check no argument passed
```

Comment:
Be sure to specify _what_ we're expecting when no argument is passed.

```suggestion
    # List all users if no argument is passed
```